webpack.config.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
   resolve: {
     extensions: ['.ts', '.tsx', '.js'],
     fallback: {
-      fs: false
+      fs: false,
+      stream: false
```

Comment:
sorry i'm not frontend expert, could you explain what excatcly you try to achieve here? avoid `steam` module resolution?

Reply:
Webpack < 5 used to include polyfills for node.js core modules by default, but clickhouse-grafana is using Webpack 5 for build.
'@grafana/data' is using core node.js module called 'stream' which in recent node.js versions doesn't require polyfilling. Nevertheless for wider node.js versions support I'll add this polyfill here instead just ignoring it.

Reply:
great thanks for explanation